docs/testing.md

```diff
@@ -125,6 +126,9 @@ and has three fields:
   `"pass"`, defining the information needed to talk to the IPMI
```

Comment:
I think we should use format of `<switches>` to describe those fields right? I see some same inconsistency in `network-driver.md` as well. And also, line 119  should say "all of them are strings".